spyder/widgets/projects/projectdialog.py

```diff
@@ -152,8 +152,9 @@ def _get_project_types(self):
 
     def select_location(self):
         """Select directory."""
-        location = getexistingdirectory(self, _("Select directory"),
-                                        self.location)
+        location = osp.normpath(getexistingdirectory(self,
+                                _("Select directory"), self.location))

```

Comment:
Per PEP 8, the indent level here should be either a 4-space hanging indent from start of the previous line and including ``self`` on the line below, or aligned with the opening ``(`` of ``getexistingdirectory``, breaking lines as necessary to stay at or under 79 characters. The former probably makes more sense here, but up to you.

Reply:
Thank you for the notes, I'm going to work on changing the slashes to forward ones and correct the pep8 issue right now.

Reply:
> I'm going to work on changing the slashes to forward ones

I'd consider waiting on that one until others weigh in, since they may or may not share that opinion and I wouldn't want you to have to do the extra work and then reverse it.

Reply:
Okay then, thanks!